### ISSUE TITLE
nested template broke the coloring

### ISSUE THREAD

**Issue by gregorypilar** [Digitalbrainstem/Javascript-Vue#2]
If I have a templete tag like this
`  <div class="d-flex flex-row table-responsive" >
                <datatable :columns="columns" :data="getData" ref="vueDatatable">
                <template scope="{ row }">
                        <tr>...`

The coloring just broke on the javascript part .
![image](https://user-images.githubusercontent.com/699270/56482322-8ec7b900-6491-11e9-92cf-10838926370a.png)

**Comment by Betanu701** [comment]
Can you recreate this with a smaller set so that you can share the entire file? I think there is something higher that may be breaking this.

**Comment by Betanu701** [comment]
Part of this has been fixed. I am going to keep it open because I need to address items inside `<template ... >`

**Comment by kyletheninja** [comment]
The problem for me in a nutshell is this, using my dropdown component  example:
```javascript
Vue.component('test', {
  template : `
  <dropdown label='My Button contents'>
    <my-actual-menu-component></my-actual-menu-component>
  </dropdown>
  `,
   beforeCreate() {
      console.log('whatever');
    }
})
```

Above everything is highlighted correctly, but as soon as I use the template tag in there like so:

```javascript
Vue.component('test', {
  template : `
  <dropdown>
    <template #btn>
      My Button Contents<i class='icon></i>
    </template>
    <my-actual-menu-component></my-actual-menu-component>
  </dropdown>
  `,
   beforeCreate() {
      console.log('whatever');
    }
})
```

The template string and everything above it is highlighted properly, but everything after the backtick enclosed string gets highlighted as if it is a string. If I remove the ` #btn` from the `<template #btn>` tag so that there are no spaces between `template` and its opening and closing angle brackets, only the template tags themselves get highlighted as strings, and the rest of the document is fine. However, in order to use slots properly we must necessarily use the name sometimes, as in this case where the default slot is already being used for the dropdown menu, not the toggle button. I should mention that `<template v-slot:btn>` does not work either. It seems to be the space, even `<template >` gives the same bug.

For now my workaround is to put the template at the bottom of the definition object, since it only messes with the highlighting after it. Thank you for this plugin by the way! This is the best way to write components, I've wanted it since I started using angularjs 5 years ago.

**Comment by Betanu701** [comment]
@kyletheninja This should be fixed now v0.3.0. Let me know if you see something wrong in the <template>.

I do not currently use templates like this ie within brackets like html code '<template></template>', so I don't have any good test cases. I removed the functionality for <template> as it was causing colorization issues. But if you are still within template: member it should function correctly. 

If I need to add the functionality back, I can. Let me know!

**Comment by Betanu701** [comment]
Closed due to no activity with a potential fix.
